esptfa_arima/Authentication/serializers.py

```
@@ -13,32 +13,77 @@ class UserSerializer(serializers.ModelSerializer):
+            "username",
+            "email",
+            "first_name",
+            "middle_name",
+            "last_name",
+            "password",
+            "acc_type",
+            "lrn",
+            "section",
+            "is_active",
+            "is_superuser",
+        ]
```

Ruff (0.14.14)

[warning] Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)